Sección 31: (Sección Completa)

CarlosJCD opened the pull request:
Avances en código de la sección #31 del curso.

CarlosJCD committed:
Video #334 - Seccion #31 - Creacion usuario Admin en la base de datos

CarlosJCD committed:
Video #335 - Seccion #31 - Hash al registrar la contraseña del usuario

CarlosJCD committed:
Video #336 - Seccion #31 - Creación ventana Inicio de sesión

CarlosJCD committed:
Video #337 - Seccion #31 - Validacion sintactica login

CarlosJCD committed:
Video #338 - Seccion #31 - Validacion logica del correo

CarlosJCD committed:
Video #339 -  Seccion #31 - Validacion logica de la contraseña

CarlosJCD committed:
Video #340 - Seccion #31 - Superglobal $_SESSION

CarlosJCD committed:
Video #341 - Seccion #31 - Control del acceso a las paginas

CarlosJCD committed:
Video #342 - Seccion #31 (FIN SECCION) - Iniciar y cerrar sesion en el navbar

LUGAMAFE reviewed (dismissed):
@CarlosJCD tus cambios del login se ven bien pero checando de nuevo la parte de actualizar la imagen de una propiedad existente notamos @CheloChidoone y yo que no funciona dejamos mas detalles en discord. Porfavor corrigue esto antes de continuar

CarlosJCD committed:
Correccion: Actualizar imagen

Correccion en la ruta donde se almacenan las imagenes de las propiedades que impedia el actualizar una imagen asociada a una propiedad

CarlosJCD committed:
Correccion: Recargar imagen

Correccion del error que no cargaba la imagen de una propiedad justo despues de ser actualizada.
